src/service/performance-impl.js

```diff
@@ -665,6 +666,9 @@ export class Performance {
       data['value'] = this.timeOrigin_ + delta;
     }
 
+    // Emit events. Used by `gulp performance`.
+    self.dispatchEvent(createCustomEvent('csi', {detail: {label, delta}}));
```

Comment:
Let's gate this on `getMode().test` so there's zero impact on prod.

Reply:
Would that work considering `gulp performance` uses dist builds?

Reply:
I think it uses `gulp dist --fortesting`, which results in `IS_DEV == true` in mode.js.

https://travis-ci.org/github/ampproject/amphtml/jobs/693959724#L380

We just need to make sure that that `window.__AMP_TEST` is true as well for `getMode().test` to return true.

Reply:
argh. gating on `getMode().test` wont work unless we also change `gulp performance` to use a locally built copy of v0 as its control. right now it depends on the latest prod build.